### PR TITLE
update documentation with MAM and minor enhancements

### PR DESCRIPTION
**Pull request by sunasing**

### CODE REVIEW

**Comment by sunasing** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/ios-install-unmanaged.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+[!INCLUDE [Microsoft 365 Defender rebranding](../../includes/microsoft-defender.md)]
+
+**Applies to:**
+- [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/p/?linkid=2154037)
+- [Microsoft 365 Defender](https://go.microsoft.com/fwlink/?linkid=2118804)
+
+> Want to experience Defender for Endpoint? [Sign up for a free trial.](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-exposedapis-abovefoldlink) 
+
+> [!NOTE]
+> Defender for Endpoint on iOS would use a VPN in order to provide the Web Protection feature. This is not a regular VPN and is a local/self-looping VPN that does not take traffic outside the device.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install-unmanaged.md]:
```suggestion
> Defender for Endpoint on iOS uses a VPN in order to provide the Web Protection feature. This is not a regular VPN and is a local/self-looping VPN that does not take traffic outside the device.
```

File: microsoft-365/security/defender-endpoint/ios-install-unmanaged.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Configure Microsoft Defender for Endpoint risk signals in app protection policy (MAM)
+
+Microsoft Defender for Endpoint can be configured to send threat signals to be used in App Protection Policies (APP, also known as MAM) on iOS/iPadOS. With this capability, you can use Microsoft Defender for Endpoint to protect access to corporate data from unenrolled devices as well.
+
+Steps to setup app protection policies with Microsoft Defender for Endpoint are as below:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install-unmanaged.md]:
```suggestion
Steps to setup app protection policies with Microsoft Defender for Endpoint are as follows:
```

File: microsoft-365/security/defender-endpoint/ios-install-unmanaged.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Set up the connection from your Microsoft Endpoint Manager tenant to Microsoft Defender for Endpoint. In [Microsoft Endpoint manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), go to **Tenant Administration** > **Connectors and tokens** > **Microsoft Defender for Endpoint** (under Cross platform) or **Endpoint Security** > **Microsoft Defender for Endpoint** (under Setup) and turn on the toggles under **App Protection Policy Settings for iOS**.
+1. Select Save. You should see **Connection status** is now set to **Enabled**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install-unmanaged.md]:
```suggestion
1. Select **Save**. You should see **Connection status** is now set to **Enabled**.
```

File: microsoft-365/security/defender-endpoint/ios-install-unmanaged.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+1. Create app protection policy: After your Microsoft Defender for Endpoint connector setup is complete, navigate to **Apps** > **App protection policies** (under Policy) to create a new policy or update an existing one.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install-unmanaged.md]:
```suggestion
1. Create the app protection policy: After your Microsoft Defender for Endpoint connector setup is complete, navigate to **Apps** > **App protection policies** (under Policy) to create a new policy or update an existing one.
```

File: microsoft-365/security/defender-endpoint/ios-install-unmanaged.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+1. Select the platform, **Apps, Data protection, Access requirements** settings that your organization requires for your policy.
+1. Under **Conditional launch** > **Device conditions**, you will find the setting **Max allowed device threat level**. This will need to be configured to either Low, Medium, High, or Secured. The actions available to you will be **Block access** or **Wipe data**. You may see an informational dialog to make sure you have your connector set up prior to this setting take effect. If your connector is already set up, you may ignore this dialog.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install-unmanaged.md]:
```suggestion
1. Under **Conditional launch** > **Device conditions**, you will find the setting **Max allowed device threat level**. This needs to be configured to either Low, Medium, High, or Secured. The actions available to you will be **Block access** or **Wipe data**. You may see an informational dialog to make sure you have your connector set up prior to this setting taking effect. If your connector is already set up, you may ignore this dialog.
```

File: microsoft-365/security/defender-endpoint/ios-install-unmanaged.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+1. Finish with Assignments and save your policy.
+
+For more details on MAM or app protection policy, see [iOS app protection policy settings](/mem/intune/apps/app-protection-policy-settings-ios).
+
+## Deploy Microsoft Defender for Endpoint for MAM or on unenrolled devices
+
+Microsoft Defender for Endpoint on iOS enables the App Protection Policy scenario and is available in the Apple app store.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install-unmanaged.md]:
```suggestion
Microsoft Defender for Endpoint on iOS enables the app protection policy scenario and is available in the Apple app store.
```

File: microsoft-365/security/defender-endpoint/ios-install-unmanaged.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+When app protection policies are configured for apps to include device risk signals from Microsoft Defender for Endpoint, user will be redirected to install Microsoft Defender for Endpoint when using such apps. Alternately, end-users can also install the latest version of the app directly from the Apple app store.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install-unmanaged.md]:
```suggestion
When app protection policies are configured for apps to include device risk signals from Microsoft Defender for Endpoint, users will be redirected to install Microsoft Defender for Endpoint when using such apps. Alternately, users can also install the latest version of the app directly from the Apple app store.
```

File: microsoft-365/security/defender-endpoint/ios-install-unmanaged.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+- [Microsoft 365 Defender](https://go.microsoft.com/fwlink/?linkid=2118804)
+
+> Want to experience Defender for Endpoint? [Sign up for a free trial.](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-exposedapis-abovefoldlink) 
+
+> [!NOTE]
+> Defender for Endpoint on iOS would use a VPN in order to provide the Web Protection feature. This is not a regular VPN and is a local/self-looping VPN that does not take traffic outside the device.
+
+## Configure Microsoft Defender for Endpoint risk signals in app protection policy (MAM)
+
+Microsoft Defender for Endpoint can be configured to send threat signals to be used in App Protection Policies (APP, also known as MAM) on iOS/iPadOS. With this capability, you can use Microsoft Defender for Endpoint to protect access to corporate data from unenrolled devices as well.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install-unmanaged.md]:
```suggestion
Microsoft Defender for Endpoint can be configured to send threat signals to be used in app protection policies (APP, also known as MAM) on iOS/iPadOS. With this capability, you can use Microsoft Defender for Endpoint to protect access to corporate data from unenrolled devices as well.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sunasing, kindly commit the minor suggested edits if you agree with them. Many thanks.

**Pull request merged**